sdks/fabric/interoperation-node-sdk/src/AssetManager.ts

```diff
@@ -666,11 +703,65 @@ const StartHTLCAssetLockListener = (
     assetId: string,
     recipientECert: string,
     lockerECert: string,
-    lockCallback: (c: Contract, d: string, t: string, i: string, r: string, l: string, v: string) => any,
+    lockCallback: (c: Contract, d: string, t: string, i: string, r: string, l: string, v: Hash, timeout: number) => any,
 ): void => {
     StartHTLCEventListener(contract, 'LockAsset', contractId, assetType, assetId, -1, recipientECert, lockerECert, lockCallback);
 }
 
+//TODO: Timed counterpart for timed AssetLockListener for reversion
+const assetLockCallbackTimed = (
```

Comment:
I'd prefer to call this `assetLockExpirationCallback` as I think that represents the function more accurately.